hw_9/functional.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[115]:
+
+
+import sys
+
+
+# In[59]:
+
+
+# sequential_map
+def sequential_map(*args):
+    container = args[-1]
+    for i in args[:len(args) - 1]:
+        container = list(map(i, container))
+    return(container)
+
+
+# In[61]:
+
+
+# consensus_filter
+def consensus_filter(*args):
+    container = args[-1]
+    for i in args[:len(args) - 1]:
+        container = list(filter(i, container))
+    return(container)
+
+
+# In[64]:
+
+
+# conditional_reduce
+def reduce(func, cont):
+    x = cont[0]
+    for i in cont[1::]:
```

Comment:
А зачем второе двоеточие, иы вроде как здесь не указываем шаг
```suggestion
    for i in cont[1:]:
```